optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -21,14 +71,59 @@ export const GraphHistory: FC<{
   const navigate = useNavigate()
 
   return (
-    <PlotHistory
-      studies={studies}
-      logScale={logScale}
-      includePruned={includePruned}
-      colorTheme={colorTheme}
-      linkURL={linkURL}
-      router={navigate}
-      selectedTrials={selectedTrials}
-    />
+    <Box
```

Comment:
The current implementation wraps the `PlotHistory` object in a box and places the filter text field within it.
As a result, the history plot on the selection page also includes the text field.
The selection page has both a graph and a trial list, so users might expect synchronization between them. What do you think of it?

<img width="1478" height="859" alt="image" src="https://github.com/user-attachments/assets/752cd315-cc1b-4b7e-9f02-46a432034875" />